docs/dev-machine-setup.md

```diff
@@ -216,4 +237,88 @@ In a nutshell, setting up a GCP account involves the following steps:
 
 ### Microsoft Azure
 
-`[TODO]`
+Another options to set up Firecracker for development purposes is to use
+a VM on Microsoft Azure (VirtulMachine), which supports nested virtualization
+and allows to run KVM. If you don't have a Microsoft (Azure) account, you can 
+find brief instructions in [document](https://azure.microsoft.com/en-us/get-started/).
+
+Here is a brief summary of steps to create such a setup (full instructions to
+set up a Ubuntu-based VM on GCE with nested KVM enablement can be found in Azure
+[documentation](https://azure.microsoft.com/en-us/blog/nested-virtualization-in-azure/)
+
+  1. Sign in for Azure CLI 
+     fllow the commands bellow , Open the LINK and paste the code, get information.
+     ```
+     $ az login 
+     ```
+  1. SET a Azure/VirtulMachines options , Notice: nested support VM SIZE!
```

Comment:
Do you mean "VirtualMachines"?

Reply:
What does "Notice: nested support VM SIZE!" mean?

Reply:
Hi @dianpopa 
> What does "Notice: nested support VM SIZE!" mean?

firecracker is necessary "/dev/kvm". so update comments where you pointed out. Thank you.